### PR TITLE
Add media usage of non-translatable blocks of shortcodes page builders

### PR DESCRIPTION
**Pull request by strategio**
https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-5834

Note: This is already implemented in the same way for metadata page builders (https://github.com/OnTheGoSystems/wpml-page-builders/commit/6fdd8ca811250e663186cf0b8998c6f84a2028a7#diff-c6b5d5c2c50413da4633cc843e7faa55R49).

### CODE REVIEW

**Comment by strategio** [comment]
@OnTheGoSystems/wpml please review.

File: tests/phpunit/tests/media/shortcodes/test-wpml-page-builders-media-shortcodes-update.php [hunk 1]
```diff
@@ -40,10 +40,12 @@ public function it_should_not_translate_if_post_is_source() {
 
 	/**
 	 * @test
+	 * @group wpmlcore-5834
 	 */
 	public function it_should_translate() {
 		$target_lang = 'fr';
 		$source_lang = 'en';
+		$original_post_id   = mt_rand( 1, 100 );
```
Comment (kagg-design) [inline tests/phpunit/tests/media/shortcodes/test-wpml-page-builders-media-shortcodes-update.php]: When Andrea reviewed my first commits, he asked not to use random numbers/strings in tests, not to guess after why tests are not running unexpectedly.  I do not insist, of course )

Comment (kagg-design) [inline tests/phpunit/tests/media/shortcodes/test-wpml-page-builders-media-shortcodes-update.php]: When Andrea reviewed my first commits, he asked not to use random numbers/strings in tests, not to guess after why tests are not running unexpectedly. I do not insist, of course )

Comment (strategio) [reply tests/phpunit/tests/media/shortcodes/test-wpml-page-builders-media-shortcodes-update.php]: Yes @kagg-design! I rarely use random strings now, but I still use random numbers. That's an old habit I need to change.

**Pull request merged**